src/ChatWindow.test.js

```diff
@@ -19,6 +21,26 @@ it("renders ChatWindow without crashing", () => {
   mount(<ChatWindow />);
 });
 
+it("that it renders correctly (snapshot testing)", () => {
```

Comment:
Test name should be read with 'it' or 'describe'

Rename to "renders correctly"

Reply:
Update the names of rest of the test cases as well.

Reply:
Done.